release_images.yml

```diff
@@ -56,3 +56,58 @@ release_images:
       example: False
       disable_sm_tag: False
       force_release: False
+  4:
```

Comment:
nit: Can we move the 1/2/3 item configuration to [.release_images_template.yml](https://github.com/aws/deep-learning-containers/blob/master/.release_images_template.yml)

Reply:
Sure, I can add a future PR to add those items into the templates. However, we still need items 1, 2, and 3 to be in this file because we need to release those images once more.